indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -151,43 +153,49 @@ private void extractTitles(Element mods, IndexDocumentBean idb) {
         }
     }
 
-    private void extractNamesAndAffiliations(Element mods, IndexDocumentBean idb, boolean splitDepartments) {
+    private void extractNames(Element mods, IndexDocumentBean idb) {
         List<Element> names = mods.getChildren("name", JDOMNamespaceUtil.MODS_V3_NS);
         List<String> creators = new ArrayList<>();
         List<String> contributors = new ArrayList<>();
+        List<String> creatorsContributors = new ArrayList<>();
 
         for (Element nameEl : names) {
             // First see if there is a display form
             String nameValue = formatName(nameEl);
 
             if (nameValue != null) {
```

Comment:
since things are getting pretty busy in this block at this point, I'd suggest doing an early skip here rather than nesting
```suggestion
            if (nameValue == null) {
                continue;
            }
```